src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -980,17 +978,19 @@ using namespace ngraph::runtime;
 void runtime::cpu::CPU_ExternalFunction::propagate_in_place_input(
     ngraph::descriptor::Output* output, std::string input_name)
 {
-    auto it = output;
-    auto propagate_further = false;
-    do
+    std::deque<ngraph::descriptor::Output*> stack;
```

Comment:
This really could've been a separate PR  :)

Reply:
Agree. But its there now and you cant unreview it.